examples/advanced_example.py

```
@@ -0,0 +1,191 @@
+    print(f"   Field names: {person.get_field_names()}")
+    print(f"   Field items: {person.get_field_items()[:2]}...")  # Show first 2
+    
+    # 7. Error handling
+    print("\n7. Error Handling:")
+    try:
+        Person("", "Smith", 25, address, contact)
+    except ValueError as e:
+        print(f"   Caught validation error: {e}")
+    
+    try:
+        invalid_response = APIResponse(999, "Invalid", timestamp="now")
```

Ruff (0.12.2)

Local variable `invalid_response` is assigned to but never used

Remove assignment to unused variable `invalid_response`

(F841)